### DISCUSSION TITLE
[Portal engine bundle] Filters in Asset Data Pool

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#11901] [Q&A / Support]
How to use filters in asset data pool if Asset Metadata Class Definition prefix is different than name ?

Seems it's not possible, since data is indexed like name.metadata_field, and query for filter is prefix.metadata_field.

Is that a bug, or should I make custom filter using event ?

**Comment by fashxp** [comment]
Hi, can you detail your data definitions a bit more. 

thx...

**Reply by Maksonis** [reply]
Hi @fashxp , sure, sorry.

This is my definition:

![image](https://user-images.githubusercontent.com/17277953/165056788-1eab6ddb-d902-410a-af6b-3797c1a42536.png)

Definition Name is AssetMetadata, and prefix is metadata. When I add and save metadata inside asset, it is saved as 

![image](https://user-images.githubusercontent.com/17277953/165057086-679bb51e-038a-43cd-abf9-2cde7a1a8e3f.png)

but it's indexed as standard_fields.AssetMetadata.title, etc.

When I add for example filter AssetMetadata.title, filter is not shown, since data is saved as metadata.title.

I hope it's clearer now.

**Reply by fashxp** [reply]
thx for sharing, will be fixed in 2.2.4
